src/components/Header/ActiveOrganizationSelector.js

```diff
@@ -0,0 +1,137 @@
+import React from 'react';
+import {connect} from 'react-redux'
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import {setActiveOrganization, fetchUser as fetchUserAction} from '../../actions/user'
+class ActiveOrganizationSelector extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            activeOrganization: null,
+            userOrganizations: [],
+            isOpen: false,
+        };
+    }
+
+    componentDidMount() {
+        const {user: {adminOrganizations, organizationMemberships, publicMemberships}, organizations, activeOrganization} = this.props;
+        if (organizations) {
+            const userOrganizationIds = [...new Set([...adminOrganizations, ...organizationMemberships, ...publicMemberships])]
+            const userOrganizations = organizations.filter(org => userOrganizationIds.includes(org.id))
+            this.setState(() => ({userOrganizations: userOrganizations}))
+            if (!this.state.activeOrganization && !!activeOrganization) {
+                const defaultOrg = userOrganizations.find((org)=> org.id === activeOrganization)
+                this.setState(() => ({activeOrganization: defaultOrg}))
+            }
+        }
+        document.addEventListener('click', this.handleClick, false);
+    }
+
```

Comment:
If the user only belongs to one organization -> the actual html isn't being rendered but the event listener is still set which causes errors when clicking anywhere on the site. Maybe the event listener should only be added if there are more than 1 organizations.
![image](https://user-images.githubusercontent.com/50320115/210511203-4c83998f-9444-4361-aae5-5640c124044e.png)